src/styles/blocks/calendar.scss

```diff
@@ -15,16 +18,16 @@
     justify-content: center;
     align-items: center;
 
-    background-color: #eee;
-    border: 1px solid #000;
+    background-color: $main-color;
+    border: 1px solid $text-color;
```

Comment:
remove variables. you use them only once

Reply:
yeah, but it would be easier to change them next time)

Especially if I would use different formats

I used to make variables for every color)